src/PositionManager.sol

```diff
@@ -83,10 +83,10 @@ contract PositionManager is IPositionManager, ERC721Permit, PoolInitializer, Mul
                 returnData[i] = _mint(params[i]);
             } else if (actions[i] == Actions.CLOSE_CURRENCY) {
                 returnData[i] = _close(params[i], sender);
-            } else if (actions[i] == Actions.BURN) {
-                // TODO: Burn will be coupled with decrease.
-                (uint256 tokenId) = abi.decode(params[i], (uint256));
-                burn(tokenId, sender);
+                // } else if (actions[i] == Actions.BURN) {
```

Comment:
didnt wanna fix burn until i got some feedback on the draft.. cause mint/burn could change significantly with this